dataflow/gemma-flex-template/Dockerfile

```diff
@@ -18,11 +18,12 @@ FROM ${SERVING_BUILD_IMAGE}
 
 WORKDIR /workspace
 
-RUN apt-get update
-RUN apt-get install -y --no-install-recommends apt-utils
-RUN apt-get install -y --no-install-recommends curl
-RUN apt-get install -y --no-install-recommends wget
-RUN apt-get install -y --no-install-recommends git
+RUN apt-get update \
```

Comment:
This can be: 

```
RUN apt-get update \
    && apt-get install -y --no-install-recommends apt-utils curl wget git
    && rm -rf /var/lib/apt/lists/*
```

Reply:
@jrmccluskey could you update this please?

Reply:
Done